index.html

```diff
@@ -29,6 +29,7 @@ <h2>Top 3 Pug Names</h2>
 	    	<li>Pugsly</li>
 	    	<li>Peggy</li>
 	    	<li>Clyde</li>
+        <li>Puggy Sue</li>
```

Comment:
This is a **great** addition, but can you *please* change the `h2` header to `Top 4 Pug Names`?